src/tables/directory.js

```diff
@@ -44,8 +44,9 @@ Directory.preEncode = function(stream) {
   this.numTables = tables.length;
   this.tables = tables;
 
-  this.searchRange = Math.floor(Math.log(this.numTables) / Math.LN2) * 16;
-  this.entrySelector = Math.floor(this.searchRange / Math.LN2);
+  let maxPowerOf2 = 2 ** Math.floor((Math.log(this.numTables) / Math.LN2));
+  this.searchRange =  maxPowerOf2 * 16;
+  this.entrySelector = Math.log(maxPowerOf2) / Math.LN2;
```

Comment:
L47 guaranteeds that L49 is always<sup>*</sup> integer


<sup>*) up to the font-impossible value of Math.log(2**750), at least. </sup>

Reply:
I think you may need to add a Babel plugin for the exponentiation operator

Reply:
I'll rewrite it to Math.pow form - eventually IE will be dead and all those can be cleaned up, until then it's not really worth asking babel to do the conversion for.